decred/tests/unit/dcr/test_txscript.py

```diff
@@ -4448,3 +4448,147 @@ def test_as_small_int():
         assert (
             res == test["want"]
         ), f'wanted {test["want"]} but got {res} for test {test["name"]}'
+
+
+def test_pay_to_pub_key_hash_script():
+    """
+    name (str): Short description of the test.
+    pkHash (ByteArray): The public key hash to pay to.
+    wantException (Exception): The exception expected if any.
+    want (ByteArray): The script that pays to the public key hash.
+    """
+    tests = [
+        dict(
+            name="ok",
+            pkHash=parseShortForm("NULL_BYTES_20"),
+            want=parseShortForm(
+                "DUP HASH160 DATA_20 NULL_BYTES_20 EQUALVERIFY CHECKSIG"
+            ),
+        ),
+        dict(
+            name="pubkey hash wrong size",
+            pkHash=parseShortForm("NULL_BYTES_21"),
+            wantException=DecredError,
+        ),
+    ]
+    for test in tests:
+        if test.get("wantException"):
+            with pytest.raises(test["wantException"]):
+                res = txscript.payToPubKeyHashScript(test["pkHash"])
+            continue
+        res = txscript.payToPubKeyHashScript(test["pkHash"])
+        assert (
+            res == test["want"]
+        ), f'wanted {test["want"].hex()} but got {res.hex()} for test {test["name"]}'
+
+
+def test_pay_to_script_hash_script():
+    """
+    name (str): Short description of the test.
+    scriptHash (ByteArray): The script hash to pay to.
+    want (ByteArray): The script that pays to the script hash.
+    """
+    res = txscript.payToScriptHashScript(parseShortForm("NULL_BYTES_20"))
+    want = parseShortForm("HASH160 DATA_20 NULL_BYTES_20 EQUAL")
+    assert res == want, f"wanted {want.hex()} but got {res.hex()}"
+
+
+def test_pay_to_pubkey_script():
+    """
+    name (str): Short description of the test.
+    serializedPubkey (ByteArray): The pubkey to pay to.
+    want (ByteArray): The script that pays to the pubkey.
+    """
+    res = txscript.payToPubKeyScript(parseShortForm("NULL_BYTES_33"))
+    want = parseShortForm("DATA_33 NULL_BYTES_33 CHECKSIG")
+    assert res == want, f"wanted {want.hex()} but got {res.hex()}"
+
+
+def test_pay_to_stake_pubkey_hash_script():
+    hash160 = parseShortForm("NULL_BYTES_20")
+    addr = addrlib.AddressPubKeyHash(hash160, testnet, crypto.STEcdsaSecp256k1)
+    """
+    name (str): Short description of the test.
+    addr (AddressPubKeyHash): An address.
+    stakeCode (int): An op code for the type of stake submission transaction.
+        SSTX or SSTXCHANGE.
+    want (ByteArray): The script that pays to the pubkey hash of the address.
+    """
```

Comment:
Using a multi-line string like this is unusual. It looks like a docstring but it's not in the docstring place. Maybe turn it into a multi-line comment?

Reply:
Been doing this for all the tests though, to document the named variables. By multi-line comment you mean to use `#` I guess? Will change for this test if it's confusing. Some of the txscript tests still used unittest too, so I guess someone could change all the other instances while updating, if that were to happen.